CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+# Solo Community Code of Conduct
+
+All members of the Solo community must abide by the following Code of Conduct,
+adapted from the [Contributor Covenant][homepage]. Only by respecting each
+other can we develop a productive, collaborative community.
+
+## Our Pledge
+
+As members, contributors, and leaders of the Solo community, we pledge to make
+participation in our community a harassment-free experience for everyone,
+regardless of age, body size, visible or invisible disability, ethnicity, sex
+characteristics, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* Post non relevant spam contents
+* Attempting to poach solo employees
```

Comment:
This doc is pretty much reuse Contributor Covenant 2.0, these 2 bullets are added